### ISSUE TITLE
Does robustness gain originate from better converge of adv training?

### ISSUE THREAD

**Issue by YonggangZhangUSTC** [facebookresearch/ImageNet-Adversarial-Training#13]
Hi, I am deeply attracted by your paper. However, here are three questions that confuse me.
1. CW loss is more powerful than cross-entropy used in your code, because attackers can decrease logit z_y(y is the label) effectively. Will it be more convincing to use CW loss when you claim your model are robust. Intuitively, the unbreakable model may partially  mean that the system is unlearnable.
2. I find that the robustness of the denoising version 'clean' trained model is not reported. I wonder how much gain denoising blocks without adv training can bring.
3. Does robustness gain originate from better converge of adv training? 
As I do not find the number mentioned in question 2, I conjecture that denoising block is beneficial for the adv training, which means that denoising blocks may improve the accuracy of adv trained model on clean images. Numbers reported in Sec. 6.3 show that adv training with denoising blocks can increase accuracy from 62.3% to 65.3%. And, Figure 6 shows that adv training with denoising blocks can increase accuracy from 39.2% to 42.6%, also about 3%. Thus is the next step making adv training converge better?
Looking forward to your reply. :-)

**Comment by ppwwyyxx** [comment]
1. We have released our models so everyone can try any attacks on our models to see if they are robust.

2. Models not trained with adversarial training will have roughly 0% accuracy under a multi-step PGD white-box attack.

3. 
> Does robustness gain originate from better converge of adv training?

> Thus is the next step making adv training converge better?

If "better" does not mean robustness, what do you mean by "better"?

> denoising blocks may improve the accuracy of adv trained model on clean images. Numbers reported in Sec. 6.3 show that adv training with denoising blocks can increase accuracy from 62.3% to 65.3%. And, Figure 6 shows that adv training with denoising blocks can increase accuracy from 39.2% to 42.6%, also about 3%. 

These are true.

**Comment by Muzammal-Naseer** [comment]
Hi,
From where can I download the pretrained models.

**Comment by xingjunm** [comment]
Hi there, I found the models in INSTRUCTIONS.md file. :)
